diff -r -u postgresql-orig/templates/_helpers.tpl postgresql/templates/_helpers.tpl
--- postgresql-orig/templates/_helpers.tpl	2020-07-02 09:48:34.000000000 +0200
+++ postgresql/templates/_helpers.tpl	2020-07-06 12:21:03.000000000 +0200
@@ -40,9 +40,9 @@
 Return the appropriate apiVersion for networkpolicy.
 */}}
 {{- define "postgresql.networkPolicy.apiVersion" -}}
-{{- if semverCompare ">=1.4-0, <1.7-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- if default "v1.18.0" .Capabilities.KubeVersion.Version | trimPrefix "v" | semverCompare ">=1.4-0, <1.7-0" -}}
 "extensions/v1beta1"
-{{- else if semverCompare "^1.7-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- else if default "v1.18.0" .Capabilities.KubeVersion.Version | trimPrefix "v" | semverCompare "^1.7-0" -}}
 "networking.k8s.io/v1"
 {{- end -}}
 {{- end -}}
@@ -395,7 +395,7 @@
 Return the appropriate apiVersion for statefulset.
 */}}
 {{- define "postgresql.statefulset.apiVersion" -}}
-{{- if semverCompare "<1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- if default "v1.18.0" .Capabilities.KubeVersion.Version | trimPrefix "v" | semverCompare "<1.14-0" -}}
 {{- print "apps/v1beta2" -}}
 {{- else -}}
 {{- print "apps/v1" -}}
@@ -444,9 +444,9 @@
 Return the appropriate apiVersion for podsecuritypolicy.
 */}}
 {{- define "podsecuritypolicy.apiVersion" -}}
-{{- if semverCompare "<1.10-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- if default "v1.18.0" .Capabilities.KubeVersion.Version | trimPrefix "v" | semverCompare "<1.10-0" -}}
 {{- print "extensions/v1beta1" -}}
 {{- else -}}
 {{- print "policy/v1beta1" -}}
 {{- end -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
